CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 3.5.1)
+project(CMaizeSuperbuild VERSION 1.0.0)
+
+include(ExternalProject)
+include(GNUInstallDirs)
+set(
+        STAGING_DIRECTORY "${CMAKE_BINARY_DIR}/stage/"
+)
+
+set(
+        INCLUDE_DIRECTORY hello_world/include
+)
+
+ExternalProject_Add(
+    "${PROJECT_NAME}_core"
+    SOURCE_DIR "${CMAKE_CURRENT_LIST_DIR}/hello_world"
+    CMAKE_ARGS -DCMAKE_CXX_COMPILER="${CMAKE_CXX_COMPILER}"
+               -DCMAKE_CXX_STANDARD="${CMAKE_CXX_STANDARD}"
+               -DCMAKE_INSTALL_PREFIX="${STAGING_DIRECTORY}"
+    CMAKE_CACHE_ARGS -DCMAKE_CXX_FLAGS:STRING="${CMAKE_CXX_FLAGS}"
+    BUILD_ALWAYS 1
+        DESTDIR=${CMAKE_BINARY_DIR}/stage
```

Comment:
This is just `STAGING_DIRECTORY`; also does `ExternalProject_Add` recognize `DESTDIR`?